### DISCUSSION TITLE
Semantic HTML examples

### DISCUSSION THREAD

**Discussion by AnthonyPAlicea** [saadeghi/daisyui#1816] [Ideas / feature requests]
Hello,

I love how Daisy UI is providing a way to use the utility classes of Tailwind while providing common UI component classes. I was curious what the feeling was on doing some updates to the HTML in the examples, where appropriate, to be more semantic/accessible.

For example, the footer component currently is:

`<footer class="footer p-10 bg-base-300 text-base-content">`
  `  <div>`
    `    <span class="footer-title">Services</span>`
    `    <a class="link link-hover">Branding</a>` 
    `    ...`
  `  </div>` 
`</footer>`

and a more semantic approach would utilize at least a `header` wrapper for what is currently `span`s. It would also make semantic sense for each column to be a `section` of the footer. For example:

`<footer class="footer p-10 bg-base-300 text-base-content">`
  `  <section`
    `    <header class="footer-title"><h2>Services</h2></header>`
    `    <a class="link link-hover">Branding</a>`
    `    ...`
  `  </section>`
`</footer>`

or even just a `<p>` instead of an `<h2>` if the desire isn't to be too specific in choices. I recently did a bunch of this kind of work for Qwik's documentation site (https://github.com/BuilderIO/qwik/pull/3745/files) and I was wondering what the feeling would be about creating a PR with some of these kinds of updates. These changes often require no CSS changes. Devs copy/paste examples, as opposed to making their own semantic/accessible, thus driving this thought.

Thanks!

**Comment by ThatXliner** [comment]
There seems to be MD files you can edit and make a PR for here: https://github.com/saadeghi/daisyui/tree/master/src/docs/src/routes/components

**Comment by saadeghi** [comment]
I agree. And I appreciate contributions.  
But now I'm working on daisyUI 3.0 which also comes with a lot of changes on the docs website. So I would suggest if you want to open a PR, please wait until daisyUI 3.0 release (probably at the end of this month)

**Comment by AnthonyPAlicea** [comment]
> I agree. And I appreciate contributions.  
> 
> But now I'm working on daisyUI 3.0 which also comes with a lot of changes on the docs website. So I would suggest if you want to open a PR, please wait until daisyUI 3.0 release (probably at the end of this month)

Sounds great, looking forward to it!
